api/v1alpha1/openapi.yaml

```diff
@@ -2005,6 +2005,36 @@ components:
           description: 'Current state of systemd units on the device.'
           items:
             $ref: '#/components/schemas/DeviceSystemdUnitStatus'
+        applications:
+          type: object
+          description: 'Current status of the device applications.'
+          items:
+            $ref: '#/components/schemas/DeviceApplicationsStatus'
+        runtime:
+          type: object
+          description: 'Current device runtime status.'
+          items:
+            $ref: '#/components/schemas/DeviceRuntimeStatus'
+        update:
+          type: object
+          description: 'Current device update status.'
+          items:
+            $ref: '#/components/schemas/DeviceUpdateStatus'
+        integrity:
+          type: object
+          description: 'Current device integrity status.'
+          items:
+            $ref: '#/components/schemas/DeviceIntegrityStatus'
+        config:
+          type: object
+          description: 'Current device config status.'
+          items:
+            $ref: '#/components/schemas/DeviceConfigStatus'
+        resources:
```

Comment:
The `runtime:`/`system:` status already factors in the status of the individual resources (in "degraded" and "error" states). I'm wondering whether we should either keep these individual resource statuses under `conditions:` like now (if we can figure out the "warn" and "alarm" watermarks) or move them under `runtime:`/`system:`?
Basically, like `applications:` which has a summary status plus list of constituent applications, we could have a system summary status plus list of constituent system states and resources.

Reply:
Makes sense can we add this to the DDD please?